major_event_log/feeds.py

```diff
@@ -8,7 +8,54 @@
 from .models import Event
 
 
-class MajorEventLogFeed(Atom1Feed):
+class PaginatedFeedTypeMixin(object):
```

Comment:
If we have to completely override `add_root_elements`, without calling `super` then we will need to rename this to be Atom specfic.

`PaginatedAtom1FeedMixin`